src/main/java/org/opensearch/performanceanalyzer/commons/metrics/AllMetrics.java

```diff
@@ -1023,13 +1043,17 @@ public static class Constants {
                     "cancellationCount";
             public static final String SEARCHBP_SEARCH_TASK_STATS_LIMITREACHEDCOUNT =
                     "limitReachedCount";
+            public static final String SEARCHBP_SEARCH_TASK_STATS_COMPLETIONCOUNT =
+                    "completionCount";
             public static final String SEARCHBP_SEARCH_TASK_STATS_RESOURCE_USAGE_TRACKER_STATS =
                     "resourceUsageTrackerStats";
             // SearchShardTaskStats
             public static final String SEARCHBP_SEARCH_SHARD_TASK_STATS_CANCELLATIONCOUNT =
                     "cancellationCount";
             public static final String SEARCHBP_SEARCH_SHARD_TASK_STATS_LIMITREACHEDCOUNT =
                     "limitReachedCount";
+            public static final String SEARCHBP_SEARCH_SHARD_TASK_STATS_COMPLETIONCOUNT =
+                    "completionCount";
```

Comment:
nit: this is in fact the same string as `SEARCHBP_SEARCH_TASK_STATS_COMPLETIONCOUNT`, do we need 2 separate constants?

Reply:
Yes, they both represent completion count at co-ordinator and shard level task completion counts.